<launch>
    <node pkg="pnp" exec="pnp_node" name="pnp_node">

        <!-- Subscribed topics -->
        <param name="/slam/odom" value="/slam/odom"/>
        <param name="/kpr/keypoints_topic" value="/kpr/keypoints_topic"/>

        <!-- Published topics -->
        <param name="/pnp/landmark_array" value="/pnp/landmark_array"/>

        <!-- camera params -->
        <param name="/camera_interface/cx" value="348.13569685"/>
        <param name="/camera_interface/cy" value="170.94638731"/>
        <param name="/camera_interface/fx" value="360.02287019"/>
        <param name="/camera_interface/fy" value="359.43717375"/>
        <param name="/camera_interface/k1" value="-0.11924763"/>
        <param name="/camera_interface/k2" value="-0.09340822"/>
        <param name="/camera_interface/p1" value="0.00170718"/>
        <param name="/camera_interface/p2" value="0.00122067"/>
        <param name="/camera_interface/k3" value="0.09476736"/>

        <!-- cone params -->
        <param name="/pnp/big_cone_dims" value="[[0,0,505], [], [], [], [], [], []]"/>
        <param name="/pnp/small_cone_dims" value="[[], [], [], [], [], [], []]"/>
        
    </node>
</launch>